### ISSUE TITLE
Uploads appear twice on website

### ISSUE THREAD

**Issue by polarbearing** [mapillary/mapillary_issues#2376]
Website:
As already mentioned in #2366, the website shows under "Uploads" every sequence after Oct 9 twice. They come with the same sequence ID, so no real duplicates. The uploads before that date appear only once.

![dup-uploads](https://cloud.githubusercontent.com/assets/8266355/20609190/8bcf1016-b289-11e6-92f6-11d81501870b.png)

**Comment by polarbearing** [comment]
Ah sorry duplicate of #2313.
